vignettes/analysing-n-grams.Rmd

```diff
@@ -27,7 +27,7 @@ for
 of articles, we only select articles from 1997 to 2017. After
 logging in/creating an account, we select unigrams and bigrams. After unzipping
 the archives to a convenient location, we start our analysis.
-
+(#1)
```

Comment:
Either explain the whole process of acquiring data from jstor/dfr, or do not explain it at all, and point the reader to the docs of jstor/dfr. Like this...not so good. the "it has to be unzipped" should be included however, I think